### DISCUSSION TITLE
I'm getting this error message, thank you for your help.

### DISCUSSION THREAD

**Discussion by ghost** [swar/Swar-Chia-Plot-Manager#995] [Q&A]
![swar_manager_err](https://user-images.githubusercontent.com/52304238/120065547-cbbcfb80-c0ac-11eb-826c-b2852915fea7.PNG)

**Comment by FlyingToto** [comment]
I think I had the same error earlier, in your config.yaml, the list of destination directory is 1 per line not all on the same line... check the second example in the config.yaml.default... 

```
destination_directory:
       - J:\Plots
       - K:\Plots
```

**Reply by ghost** [reply]
Solved!!  Thank you.
